dotnet/Services/ProductReviewService.cs

```diff
@@ -331,7 +331,7 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
             AppSettings settings = await GetAppSettings();
             if (settings.RequireApproval)
             {
-                searchQuery = $"{searchQuery}&approved=true";
+                searchQuery = $"{searchQuery} AND approved=true";
```

Comment:
The _approved=true_ comes after the {searchQuery}, maybe you should update it as well since it is using the "&" still.

Reply:
This will run only on the store front as the admin never uses the `locale` and the store front never uses the `_keyword` to search for a review, (at least for now).